build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -332,7 +332,14 @@ service ContentAddressableStorage {
 // The query may include a particular `instance_name`, in which case the values
 // returned will pertain to that instance.
 service Capabilities {
-  // GetCapabilities returns the server capabilities configuration.
+  // GetCapabilities returns the server capabilities configuration of the
+  // remote endpoint.
+  // Only the capabilities of the services supported by the endpoint will
+  // be returned. For example:
```

Comment:
Is 'For example' necessary here? Couldn't we simply specify an exhaustive list?

Reply:
Hmm, I guess this is already an exhaustive list :-)